src/components/lib/esmap.js

```diff
@@ -75,10 +76,20 @@ function renderEdges(g, data, ref, layerId, options) {
   const doEdgeMouseOver = (event, d) => {
     var start = new Date();
     var thisEdge = d3.select(event.target)
+    var thisEdgeClassName = [...event.target.classList].find(classListItem => /edge-(az|za)-(.*)/.test(classListItem));
```

Comment:
This could use a refactor I think. We determine whether this is a AZ or ZA edge below, which seems more reliable. The reverse edge can be calculated from that I think. We also have control over the edge classes which should allow for a cleaner mechanism here overall. Doing regex match on class list feels very breakable.